ApplicationLibCode/Application/RiaDefines.cpp

```diff
@@ -374,3 +374,11 @@ std::vector<double> RiaDefines::viewScaleOptions()
 {
     return { 0.005, 0.01, 0.05, 0.1, 0.2, 0.5, 1.0, 2.0, 3.0, 4.0, 5.0, 10.0, 15.0, 20.0, 50.0 };
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RiaDefines::betaFeaturePostfix()
+{
+    return " [BETA]";
```

Comment:
Is BETA a term end users will understand?

Reply:
Equinor agrees to use the term `BETA.`